recipes/bte/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "bte" %}
+{% set version = "0.8.0" %}
+{% set sha256 = "481b9d43cbc41b6b451856b185e082a2e7d03bb2613c362811a750f5e935f512" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jmcbroome/BTE/releases/download/v0.8.0/bte.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py < 38]
+
+requirements:
+  build:
+    - make
+    - setuptools
+    - {{ compiler("cxx") }}
+  host:
+    - cython
+    - python {{ python }}
+    - boost-cpp=1.73.0
+    - protobuf=3.19
+    - tbb-devel=2019.0   
```

Comment:
We have project-wide pins for boost-cpp and tbb-devel. Can we use those so this is compatible with other packages in the same environment?

Reply:
The boost version may be able to be changed (I haven't tested it) but I do have to have the 2019 version of tbb-devel (its an explicit dependency of the MAT module BTE extends). I'm new to conda package development, so maybe I'm not following how exactly pinning dependencies works, but it looks like you pin against the 2020 version of tbb-devel (because that's what conda forge uses in their config?). Are there steps I can take to resolve this and get this confirmed, or will it not be possible to distribute here because of this versioning issue?

Reply:
If there's no way around needing to support this pinning I could look into manually including the tbb source code needed as a submodule and building/including it separately, but I'd rather be able to use conda to handle my dependencies.

Reply:
If newer versions of TBB aren't supported then we'll just go with things as they are. I don't know how much TBB changes between versions.